pkg/storage/node_test.go

```diff
@@ -86,7 +86,7 @@ var _ = Describe("Node storage", func() {
 
 		err := testutil.CollectAndCompare(pointsStored, strings.NewReader(`
 		`), "metrics_server_storage_points")
-		Expect(err).NotTo(HaveOccurred())
+		Expect(err.Error()).To(Equal("expected metric name(s) not found: [metrics_server_storage_points]"))
```

Comment:
why does this change with the dependency bump? The metrics_server should still be available

Reply:
This is because the comparison of the dependent library has changed, which does not affect the functionality. When the value is null, the comparison will now return an error
https://github.com/prometheus/client_golang/blob/ac114f315e2b5f4bd0fce2830fbb6d28d348eb02/prometheus/testutil/testutil.go#L278

Reply:
This comparison function `CollectAndCompare` we only use in testing

Reply:
/cc @dgrisonnet

Reply:
I see, thank you for looking into it.